We do not have smokeshow packaged.  It is not needed to run tests.

--- annotated_doc-0.0.5/pyproject.toml.orig
+++ annotated_doc-0.0.5/pyproject.toml
@@ -51,7 +51,6 @@
 ]
 tests = [
     "pytest>=8.3.5",
-    "smokeshow>=0.5.0",
     "typer>=0.16.1",
 ]
 
